src/iotjs.c

```diff
@@ -224,9 +247,13 @@ int iotjs_entry(int argc, char** argv) {
   iotjs_environment_set_loop(env, uv_default_loop());
 
   // set parser dump file
-  char dump_path[20];
-  snprintf(dump_path, 20, "/tmp/iotjs.%zu", (size_t)getpid());
```

Comment:
Macro is better than magic number, especially if appear more than once.

Reply:
What do you mean by the magic number?

Reply:
I mean 20. It's a good habit. If more than one magic number are far from each other, it's easy to forget sync all if want change value.
But your code is OK here.

Reply:
Yea, I have used `sizeof` instead of the numbers :)